users/views.py

```diff
@@ -24,3 +24,10 @@ def get_serializer_class(self):
         if self.request.user == self.get_object():
             return UserPrivateSerializer
         return UserPublicSerializer
+
+
+class SearchUserView(generics.ListAPIView):
```

Comment:
Is it possible to unify this view and `RetrieveUpdateUserProfileView`?

Reply:
I tried, but I have an error that `RetrieveUpdateUserProfileView` can't be called without a URL keyword argument named `"pk"`